dev/system-test/firestore.ts

```diff
@@ -287,6 +287,20 @@ describe('DocumentReference class', () => {
         });
   });
 
+  it('supports numericAdd()', () => {
```

Comment:
In the Server SDK, all this behavior is driven by the backend. Since there is no special handling here, I don't think it makes sense to add these tests. Note that we do have these tests in the Mobile SDKs.